packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -120,12 +100,34 @@ export const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
           mac: 'Enter',
         },
         exec: () => {
-          onApply();
+          onApply?.();
```

Comment:
👀 What does this syntax do? calls an optional function?

Reply:
Yep! It's optional chaining syntax for calling functions